suite-native/module-dev-utils/src/screens/DevUtilsScreen.tsx

```diff
@@ -12,124 +10,40 @@ import {
     ScreenHeader,
 } from '@suite-native/navigation';
 import { purgeStorage, useStoragePersistor } from '@suite-native/storage';
-import { selectLogs } from '@suite-common/logger';
-import { useCopyToClipboard } from '@suite-native/helpers';
 
 import { BuildInfo } from '../components/BuildInfo';
-
-const DevCheckBoxListItem = ({
-    title,
-    onPress,
-    isChecked,
-}: {
-    title: string;
-    onPress: () => void;
-    isChecked: boolean;
-}) => (
-    <TouchableOpacity onPress={onPress}>
-        <Box
-            flexDirection="row"
-            justifyContent="space-between"
-            alignItems="center"
-            paddingVertical="small"
-        >
-            <Text variant="body">{title}</Text>
-            <CheckBox isChecked={isChecked} onChange={onPress} />
-        </Box>
-    </TouchableOpacity>
-);
-
-const CopyLogsButton = () => {
-    const logs = useSelector(selectLogs);
-    const copyToClipboard = useCopyToClipboard();
-
-    const handleCopy = async () => {
-        await copyToClipboard(JSON.stringify(logs), 'Logs copied');
-    };
-
-    return <Button onPress={handleCopy}>Copy logs</Button>;
-};
-
-const Logs = () => {
-    const logs = useSelector(selectLogs);
-    return (
-        <Box>
-            <Text>{JSON.stringify(logs)}</Text>
-        </Box>
-    );
-};
+import { Logs } from '../components/Logs';
+import { RenderingUtils } from '../components/RenderingUtils';
+import { ProductionDevInfo } from '../components/ProductionDevInfo';
 
 export const DevUtilsScreen = ({
     navigation,
 }: StackProps<DevUtilsStackParamList, DevUtilsStackRoutes.DevUtils>) => {
     const persistor = useStoragePersistor();
-    const {
-        toggleFlashOnRerender,
-        toggleRerenderCount,
-        isFlashOnRerenderEnabled,
-        isRerenderCountEnabled,
-    } = useDebugView();
-    const [areLogsVisible, setAreLogsVisible] = useState(false);
 
-    const handleResetStorage = () => {
-        purgeStorage(persistor);
+    const handleResetStorage = async () => {
+        await purgeStorage(persistor);
```

Comment:
there is no reason to do this since you are not doing anything after

Reply:
Removed